Formula/dnsviz.rb

```diff
@@ -0,0 +1,46 @@
jxkiG94xhJCbhZcmJq3DEc27I25amDYik9NkYROgfmIA9+UcsmJ0tFJ0u4OBS6JN9iMomPOXcPFz/ZF3wGdCV1Jcx3Kj3ytOX2XccjAMF4kvbwx2/U+pEm1xBDebTTOXC8biZ6gHF3AVRFgAAKQIAAACAAAAA\", \"msg_size\": 525, \"time_elapsed\": 25, \"history\": []}}}}, {\"qname\": \".\", \"qclass\": \"IN\", \"qtype\": \"SOA\", \"options\": {\"flags\": 256, \"edns_version\": 0, \"edns_max_udp_payload\": 4096, \"edns_flags\": 32768, \"edns_options\": [], \"tcp\": false}, \"responses\": {\"127.0.0.1\": {\"127.0.0.1\": {\"message\": \"rQmBoAABAAIAAAABAAAGAAEAAAYAAQAATiEAQAFhDHJvb3Qtc2VydmVycwNuZXQABW5zdGxkDHZlcmlzaWduLWdycwNjb20AeCtmeAAABwgAAAOEAAk6gAABUYAAAC4AAQAATiEBEwAGCAAAAVGAWCqWUFgZZMCZewAlgFTcjso0WCaN8gG1M1bAGce8KhyDRZsYGIHzMpJmSZyB6TIJMuFq+/a9VDqDXcUq5F3EVjppXN8v8yLYMUoQmW3JmkXMlH/70ID2AImRVciDFhQO48WY4wgq1lhRe/nxmp7Qxvk8b2G4BQma4Qkel4f/UAaFogDpv7DtTfvQydDi8ZcduN2MbDiQqZZCV/CPGf2ekJZX7YRrC3j+VfLfyck5VFv2xzzP7aWxtpzhk7L4xeGvDsmbaiU0xNb+oq+mhw7SU1AZLsQUhXe3v1uxFlhMc0au6BSskHwDIsaNX/cS+8NM1DsNU7neZNzHteekzKcHpwcJhfJx+GS/XtKAAAApAgAAAIAAAAA=\", \"msg_size\": 389, \"time_elapsed\": 26, \"history\": []}}}}, {\"qname\": \".\", \"qclass\": \"IN\", \"qtype\": \"DNSKEY\", \"options\": {\"flags\": 256, \"edns_version\": 0, \"edns_max_udp_payload\": 512, \"edns_flags\": 32768, \"edns_options\": [], \"tcp\": false}, \"responses\": {\"127.0.0.1\": {\"127.0.0.1\": {\"message\": \"Z/ODgAABAAAAAAABAAAwAAEAACkCAAAAgAAAAA==\", \"msg_size\": 28, \"time_elapsed\": 38, \"history\": []}}}}, {\"qname\": \".\", \"qclass\": \"IN\", \"qtype\": \"DNSKEY\", \"options\": {\"flags\": 256, \"edns_version\": 0, \"edns_max_udp_payload\": 4096, \"edns_flags\": 32768, \"edns_options\": [], \"tcp\": false}, \"responses\": {\"127.0.0.1\": {\"127.0.0.1\": {\"message\": \"ex6BoAABAAMAAAABAAAwAAEAADAAAQAAM3wBCAEBAwgDAQABqAAgqVVmukLohruATNqE5H71bb167GEmFVUs7JBtIRbQ7yBwKMUVVBRN/q/nx8uPAF3RgjQTOsBxCoEYLOH9FK0ig7yDQ1+d8vYxMlGTGhdt8NpR5U9C5gSGDfs1lYAlD1WcxUPE/9Ucvj3oz9BnGSN/n8R+5ynaBoNfpFLoJemhjrwuy89WNHRlLDPPVqkDO8312XMSF5fsgIkEG24DobctCnNbmE4DaHMJMyMk8nwtuoXp2xXoOgFDOC6XSwYhwY5iXs7JB1d9nnut6VJBqB676KkB1NMnbkCxFMCi5vw40ZwuaqsCZEsoE/V1/CFgHg3uSc2e6WpDED5STWKHPQAAMAABAAAzfAEIAQADCAMBAAGG4p2rhwPaFG+GhjbSSK3qchqGMtDi9Na7yIb6N0VjfwofKdguGRJKb1GJMe+K/SaYKzrIyjr/iUHSrU9rvAMRo1KB/MfB3aQ4jFe156ViaPcAMZRbQ7WzJz8EUN3VrE9aeZsqxnJuxBSfY+ypogBBuYRrHr2anJ6jDlvUBZJqf8zNExWF2FqugLAKH9rmai1MnfpxyFtExrGhzM3EzXwPFsPjmFe0eG3Ef/HnQhKOjgIfY+PEdYUbbjtXtjEHZ5CE3PQyZYweAqtYECsL/5Lu5gL5hblpEaLTGSBTd7w4aZqZsuzNAtRB0kQID/h5G5S0UCjj58CP1ZLCRFOH5p4ZAAAuAAEAADN8ARMAMAgAAAKjAFgqT/9YFomASlwALPuldf5oWFdSHSTPYL5WvrvwJTElxY6LTEw2Cit0JOcVAbZG6LLCmlpCJ55Ngf/sdE4UXUPJ/m6CFRYT+aAePvEWrjRPGGX64V82oCeCPyAqD4XHd3CIQi3LBYk8ZbEktyvBX+VS16rbSEQib7xNYvohtiJ0dRiw/wjr6YVF8xUdYO1vvXPYOGXISYwW4vDiKAuyLDGuoLRh/F9GZQxBPwv6Bmx8/JfNCfIygbnZ/8qIZUsFH68DPbAHPBqwR1GP+haAa6vQPhXwn4p+Vci7rYNzfPzdQfDNWsQ+8ur8xxSdanAZcZRrytaidLtIQx4DeGANdwmNjnAn8ZSg6q8etQAAKQIAAACAAAAA\", \"msg_size\": 864, \"time_elapsed\": 30, \"history\": []}}}}]}, \"_meta._dnsviz.\": {\"version\": 1.1, \"names\": [\".\"]}}"
```

Comment:
Indent by two characters, split this into multiple lines and don't escape the `"`. I'd suggest using `jq` to pretty-print the output for here.